ansible/templates/minigraph_meta.j2

```diff
@@ -204,7 +204,7 @@
           <a:DeviceProperty>
             <a:Name>SubRole</a:Name>
             <a:Reference i:nil="true"/>
-            <a:Value>Fabric</a:Value>
+            <a:Value>BackEnd</a:Value>
```

Comment:
@jfeng-arista, subrole backend is valid of chassis-packet chassis.

Reply:
I reverted this back to Fabric, and handle this for the test in a new PR at https://github.com/sonic-net/sonic-utilities/pull/2717

Please merge https://github.com/sonic-net/sonic-utilities/pull/2717 first.